deployment/docker-compose.prod.yml

```
@@ -104,6 +104,61 @@ services:
     networks:
       - xyne
 
+  keycloak:
+    image: quay.io/keycloak/keycloak:latest
+    container_name: xyne-keycloak-prod
+    environment:
+      # Admin credentials - Change these in production!
+      - KEYCLOAK_ADMIN=${KEYCLOAK_ADMIN:-admin}
+      - KEYCLOAK_ADMIN_PASSWORD=${KEYCLOAK_ADMIN_PASSWORD:-admin}
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)